lib/dmv.rb

```diff
@@ -8,9 +9,52 @@ def add_facility(facility)
     @facilities << facility
   end
 
-  def facilities_offering_service(service)
-    @facilities.find do |facility|
-      facility.services.include?(service)
+  def facilities_offering_service(*service)
+    arr =[]
+    service.each do |i|  
+    @facilities.each do |facility|
+      if facility.services.include?(i)
+        arr << facility
+      end
+     end
     end
+    arr
   end
```

Comment:
This will work! I think you've got a lot of extra code that you also don't need though. We might be able to get this working with a `find_all`. I'm thinking something like: 

```ruby
  def facilities_offering_service(service)
    @facilities.find_all do |facility|
      facility.services.include?(service)
    end
  end
```